sim/paladin/paladin.go

```diff
@@ -149,6 +156,33 @@ func (paladin *Paladin) ResetPrimarySeal(primarySeal proto.PaladinSeal) {
 	paladin.primarySeal = paladin.getPrimarySealSpell(primarySeal)
 }
 
+func (paladin *Paladin) registerStopAttackMacros() {
+
+	if paladin.divineStorm != nil && paladin.Options.IsUsingDivineStormStopAttack != paladin.divineStorm.Flags.Matches(core.SpellFlagBatchStopAttackMacro) {
+		paladin.divineStorm.Flags = paladin.divineStorm.Flags ^ core.SpellFlagBatchStopAttackMacro
```

Comment:
I'm not the best at bitwise operators so correct me if I'm wrong but I believe for all of these you can just do
```
paladin.X.Flags |= core.SpellFlagBatchStopAttackMacro
```

Reply:
simplified to using |=